pig/src/main/java/com/mongodb/hadoop/pig/MongoLoader.java

```diff
@@ -129,7 +129,7 @@ public void prepareToRead(final RecordReader reader, final PigSplit split) throw
                 boolean include = (Boolean) entry.getValue();
                 // Add the name of the outer-level field if this is a nested
                 // field. Pig will take care of pulling out the inner field.
-                String key = StringUtils.split(entry.getKey(), '.')[0];
+                String key = StringUtils.split(entry.getKey(), '\\', '.')[0];
```

Comment:
@motymichaely In hadoop v1 there is no method with 2 parameters.  In hadoop v2 it appears having '\\' passed to the 3-parameter method.